tpp-app/src/services/utils/helpers.js

```diff
@@ -16,3 +17,52 @@ const initializeEmptyYear = (yearNumber) => {
 
   return year
 }
+
+/**
+ * Check if the date, month, year combination is a valid date.
+ * @param day number (1st day of month = 1)
+ * @param month number (January = 1)
+ * @param year number
+ * @return True or False boolean
+ */
+export const isValidDate = (day, month, year) => {
```

Comment:
can we add a check in it that ensures the date is not in the future? Since we don't allow modifications to future dates

Reply:
Done!